plugins/ms-vscode.typescript/1.0.0/meta.yaml

```diff
@@ -0,0 +1,13 @@
+id: ms-vscode.typescript
+version: 1.0.0
+type: VS Code extension
+name: Typescript
+title: Typescript language features
+description: Typescript language features
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+url: https://github.com/tolusha/ms-code.typescript/releases/download/1.0.0/che-typescript-language.vsix
```

Comment:
Do we have any plan to move this plugin from `tolusha` to `che-incubator`?

Reply:
I don't mind. Should we?

Reply:
@garagatyi @l0rd WDYT?

Reply:
Yes! We should not keep this in our personal repos

Reply:
But this should not block this PR in any case